CONFIG.md

```diff
@@ -0,0 +1,200 @@
+>  # :warning: This page does not describe all of **ROFI**'s configuration options, just the most common usecase. For the full configuration options, check the manpages.
+
+<br />
+
+ 
+## :exclamation: Upgrade from old configuration format to new :exclamation:
+
+In release after '2 Jan 2020' you can convert to the new configuration format by using the following command:
+
+```bash 
+rofi -upgrade-config
+```
+
+This will create `~/.config/rofi/config.rasi` (or the default location for your system) with the previously configured options.
+
+From 1.7.0 the old configuration format is no longer supported and the `-upgrade-config` option is no longer available.
+
+<br />
+
+# Where does the configuration live
+
+Rofi's configurations, custom themes live in `${XDG_CONFIG_HOME}/rofi/`, on most systems this is `~/.config/rofi/`.
+
+The name of the main configuration file is `config.rasi`. (`~/.config/rofi/config.rasi`).
+
+# Create an empty configuration file
+
+Open `~/.config/rofi/config.rasi` in your favorite text editor and add the following block:
+
+```css
+configuration {
+
+}
+```
+You can now set the options in the `configuration` block. 
+
+# Create a configuration file from current setup
+
+If you do not want to start from scratch, or want to migrate from older configuration format, you can get tell rofi to dumps it configuration:
+
+```bash
+rofi -dump-config > ~/.config/rofi/config.rasi
+```
+
+This will have all the possible settings and their current value.
+If a value is the default value, the entry will be commented.
+
+For example:
+```css
+configuration {               
+/*  modi: "window,run,ssh,drun";*/
+/*  font: "mono 12";*/
+/*  location: 0;*/
+/*  yoffset: 0;*/
+/*  xoffset: 0;*/
```

Comment:
yoffset/xoffset  font, location are removed in the current version and should probably be removed.